GIFrameworkMaps.Web/Views/ManagementTheme/Index.cshtml

```diff
@@ -0,0 +1,39 @@
+﻿@model List<GIFrameworkMaps.Data.Models.Theme>
+@{
+    ViewData["Title"] = "Themes";
+}
+
+<partial name="ManagementPartials/BackToManagementHome" />
+
+<h1>@ViewData["Title"]</h1>
+<p class="lead">Create, edit and delete themes</p>
+
+<a href="@Url.Action("Create")" class="btn btn-primary"><i class="bi bi-plus-circle"></i> Add new theme</a>
+
+@if (Model != null)
+{
+    <table class="table table-bordered table-striped mt-2">
+        <thead>
+            <tr>
+                <th>ID</th><th>Name</th>
+            </tr>
+        </thead>
+        <tbody>
+            @foreach (GIFrameworkMaps.Data.Models.Theme theme in Model.OrderBy(a => a.Id))
+            {
+                <tr>
+                    <td>@theme.Id</td>
+                    <td>@Html.ActionLink(theme.Name, "Edit", new { id = theme.Id })</td>
```

Comment:
Since we've got some space, we could include the primary colour as a field here (maybe even with a little coloured badge with the actual colour in it)

Reply:
Badges added